tests/src/test/scala/com/nvidia/spark/rapids/AdaptiveQueryExecSuite.scala

```diff
@@ -97,6 +97,24 @@ class AdaptiveQueryExecSuite
     collectWithSubqueries(plan)(ShimLoader.getSparkShims.reusedExchangeExecPfn)
   }
 
+  test("get row counts from executed shuffle query stages") {
+    assumeSpark301orLater
+
+    skewJoinTest { spark =>
+      val (_, innerAdaptivePlan) = runAdaptiveAndVerifyResult(
+        spark,
+        "SELECT * FROM skewData1 join skewData2 ON key1 = key2")
+      val innerSmj = findTopLevelGpuShuffleHashJoin(innerAdaptivePlan)
+      val shuffleExchanges = ShimLoader.getSparkShims
+          .findOperators(innerAdaptivePlan, _.isInstanceOf[ShuffleQueryStageExec])
+          .map(_.asInstanceOf[ShuffleQueryStageExec])
+      assert(shuffleExchanges.nonEmpty)
```

Comment:
nit: maybe add a message to the assert  for  a more descriptive exception

Reply:
We haven't generally been using assertion messages. I have updated the assertion to make it more specific though.